pkg/stub/handler.go

```diff
@@ -606,23 +607,28 @@ func (h *Handler) CreateDefaultResourceIfNeeded(srcfg *v1alpha1.SamplesResource)
 	return srcfg, nil
 }
 
-func (h *Handler) WaitingForCredential(srcfg *v1alpha1.SamplesResource) error {
+func (h *Handler) WaitingForCredential(srcfg *v1alpha1.SamplesResource) (bool, error) {
```

Comment:
godoc..i'm not clear on what the bool return value represents?  I assumed it meant "we are waiting for the credential"(true/false)" but you use it as "returnNil" below?

Reply:
Yep will tie in with the comment I noted above